actors/builtin/market/policy.go

```diff
@@ -20,8 +23,19 @@ func dealPricePerEpochBounds(size abi.PaddedPieceSize, duration abi.ChainEpoch)
 	return abi.NewTokenAmount(0), abi.TotalFilecoin // PARAM_FINISH
 }
 
-func dealProviderCollateralBounds(pieceSize abi.PaddedPieceSize, duration abi.ChainEpoch) (min abi.TokenAmount, max abi.TokenAmount) {
-	return abi.NewTokenAmount(0), abi.TotalFilecoin // PARAM_FINISH
+func dealProviderCollateralBounds(pieceSize abi.PaddedPieceSize, verified bool, networkQAPower, baselinePower abi.StoragePower, networkCirculatingSupply abi.TokenAmount) (min abi.TokenAmount, max abi.TokenAmount) {
```

Comment:
While you're here, export all these functions so they can be used by clients & nodes for providing this collateral, pricing etc.